dashboard/test/ui/features/manage_assets.feature

```diff
@@ -1,6 +1,8 @@
 @no_mobile
 Feature: Manage Assets
 
+  # Skip on Safari and IE. MediaRecorder is not supported so we are removing the record button.
+  @no_safari_yosemite @no_ie
```

Comment:
Do we want Eyes tests still, to verify there _isn't_ a "Record Audio" button visible?

Reply:
Yep, I'm working on that in another PR